src/models/project/project.ts

```diff
@@ -9,9 +9,16 @@ export interface IProject {
 }
 
 export type ProjectProps =
```

Comment:
some of the keys are common in obth cases. Can we initialize the monly once?

Reply:
done